lib/simctl.js

```diff
@@ -732,6 +732,57 @@ async function getDeviceTypes () {
   }
 }
 
+
+/**
+ * Get the full list of runtimes, devicetypes, devices and pairs in JSON
+ *
+ * @return {Array<string>} List of the device types, runtimes devices and pairs JSO
+ * @throws {Error} If the corresponding simctl command fails
+ */
+async function getSimctlList () {
+  try {
+    const {stdout} = await simExec('list', 0, ['-j']);
```

Comment:
Nit: I'd rather it be `--json` instead of `-j`. I like having the full names for command line operations for readability sake.